### DISCUSSION TITLE
[Suggestion] Random Teleport around Coordinates

### DISCUSSION THREAD

**Discussion by Drc-DEV** [Zrips/CMI#1134] [Enhancement]
New Command suggestion: /cmi rtpos (playerName) (worldName) (Radius) (x) (y) (z) 
If radius is not defined, it should default to a set number, for example 50
If x y and z are not set, they should default to the world spawn coordinates

This command acts like /cmi rt, but istead of using pre-defined coordinates, the user can include custom coordinates in order to randomly teleport a player around the set coordinates in a custom radius in the set world.

Why is it useful? In order to get rid of the "Random Teleport" plugins which are quite often buggy and/or often teleport to unsafe coordinates, killing the player and making the server owners cry.

Example Usage: Arena random teleport, command attached to a pressureplate that teleports the player randomly inside the arena.

Why can't I just use the default /cmi rt? Because then I am limited to using the randomteleport only in one world at the time, while a lot of servers have less worlds than "warps" and "arenas" and such.

**Comment by sandwally** [comment]
With /tppos you can add -rng:[range] at the set location.
But I don't know how to exclude players from getting on top of buildings and so on.
Guess you can use the new placeholder where you can run math and from different outcome you can make that number be used to teleport a player on a set "random" location inside the arena.
